kubetest2-aks/deployer/up.go

```diff
@@ -416,7 +416,7 @@ func (d *deployer) waitForClusterUp() error {
 	if err != nil {
 		return fmt.Errorf("failed to new managed cluster client with sub ID %q: %v", subscriptionID, err)
 	}
-	err = wait.PollImmediate(10*time.Second, 10*time.Minute, func() (done bool, err error) {
```

Comment:
We now create and wait until it's done in L304, and the timeout is determined by L297.
waitForClusterUp is actually not called after the cluster is created successfully or failed/timed out.